routes/web.php

```diff
@@ -11,33 +22,58 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::get(
+    '/', function () {
+        return view('welcome');
+    }
+);
 
 Auth::routes();
 
 Route::get('/home', 'HomeController@index')->name('home');
 
-Route::middleware('auth')->group(function () {
-    // Route Dashboard
-    Route::get('/', 'DashboardController@index');
+Route::middleware('auth')
+    ->group(
+        function () {
+            // Route Dashboard
+            Route::get('/', 'DashboardController@index');
```

Comment:
Why modify item not related to current work ?

Reply:
I have edited the content